regression-tests/msal-node/client-credential/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "client-credential",
+  "version": "1.0.0",
+  "description": "Client Credential Regression Test",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@azure/msal-node": "^1.18.0",
```

Comment:
v1?

Reply:
We should probably include this in workspaces so you pull the local versions of msal-node (and also remove package-lock)